packages/story-editor/src/components/library/test/mediaElement.js

```diff
@@ -187,6 +192,14 @@ describe('MediaElement', () => {
   });
 
   it("should not render dropdown menu's more icon for not uploaded image", () => {
+    useLocalMedia.mockReturnValue({
```

Comment:
It seems that this is already done in `beforeEach(() => {` above, wouldn't it work for all cases?

Reply:
No, I am changing the value of the mocks to return true. I made a change that makes this cleaner. 
https://github.com/google/web-stories-wp/pull/9785/commits/833a39d9135b1152f6ddea3c3b7f712aecb08e67